### DISCUSSION TITLE
Is there a way for fooocus to create different facial expressions?

### DISCUSSION THREAD

**Discussion by lgs777** [lllyasviel/Fooocus#2309] [Q&A] [answered]
![스크린샷 2024-02-20 200918](https://github.com/lllyasviel/Fooocus/assets/5195460/c7b86659-9464-416d-867b-d0c35037e19e)

I tried to make the facial expressions as similar as possible with pyracanny, but failed. What would be the process to create a wide range of facial expressions like in the photo? Any advice would be appreciated.

**Comment by ILikeToasters** [comment, marked as answer]
Using a similar method to how this video shows how to different angles but instead with expressions I was able to do it. I didn't spend much time so I am sure I could have done better. Best way I can think of. https://youtu.be/MntZa4qLwn8
![2024-02-20_14-55-43_8812](https://github.com/lllyasviel/Fooocus/assets/4357657/79321005-dea9-49ed-89b9-7e67cdb74956)

**Reply by lgs777** [reply]
Thank you for your response. I already generated the same face using this method, but I couldn't get a wide range of facial expressions. The facial expressions are still awkward and canny seems to be able to create a similar appearance but not fully express the facial expressions. 

https://www.youtube.com/watch?v=Tzt_RevMgnA

It would be great if Fooocus supported this feature.

**Comment by tbuyle** [comment]
I was able to generate different emotion one by one, using Faceswap and prompting for the emotion within ( )

![image](https://github.com/lllyasviel/Fooocus/assets/467350/1c095f3f-3e43-49b8-86d2-8c9dbac2af0e)
A (smilling) man wearing a blue jeans jacket, (large smile), Busy street in Chicago, short hair and mustache

![image](https://github.com/lllyasviel/Fooocus/assets/467350/5ad25999-c632-492b-8e7d-397fa19485d3)
A (sad) man wearing a black suit, crying face, rainy day at the cemetery, short hair and mustache, graveyard

...
![image](https://github.com/lllyasviel/Fooocus/assets/467350/6bee8ea2-c6a4-4fe5-9428-6970e5651b4e)
![image](https://github.com/lllyasviel/Fooocus/assets/467350/fe5463f9-119d-457e-a79f-5612c5eb25d1)
![image](https://github.com/lllyasviel/Fooocus/assets/467350/e0d0ec61-09e2-4e83-bd65-1e059e407435)
![image](https://github.com/lllyasviel/Fooocus/assets/467350/d32be14c-0ece-470c-81d1-ad01c19fa76b)
